.gitignore

```diff
@@ -24,5 +24,6 @@ pyvenv.cfg
 htmlcov/
 *supply_chain_*/
 examples/supply_chain/docker-compose.yml
+examples/rl/config.yml
```

Comment:
?

Reply:
@ysqyang

Reply:
The file contains local paths that should not show up in a public repo. Users are asked to refer to maro/rl/workflows/config/template.yml to create their own config files for running distributed workflows. For single-threaded experiments, you can use main.py under examples/rl and change settings in there.